### ISSUE TITLE
Does Tart support running multiple image instances simultaneously?

### ISSUE THREAD

**Issue by alexey-tereshenkov-oxb** [cirruslabs/tart#200]
Running `tart run monterey-base` starts a GUI application with usable OS inside. 

Running `tart run monterey-base` in another terminal opens another GUI application that has only black screen and is not usable.

The `tart ip monterey-base` command shows only a single IP (even though it looks like there are two instances of the VM running).

---

Does Tart support running multiple VM instances in parallel? I use `cirrus` cli which of course can run multiple builds in parallel, but I wonder if this can be achieved without using `cirrus` purely using Tart (with / without graphics)?

**Comment by fkorotkov** [comment]
Yes, you can but you need to clone the VM to a temporary one. Behaviour in your use case is not defined.

**Comment by alexey-tereshenkov-oxb** [comment]
I see, thanks! 

To avoid creating a new issue, is there a cleaner way to stop a running VM instance than to kill a process based on PID? Sending `sudo shutdown -h now` (over ssh) doesn't seem to stop the `tart run` process (even though you can't any longer ssh into the machine).

**Comment by edigaryev** [comment]
> To avoid creating a new issue, is there a cleaner way to stop a running VM instance than to kill a process based on PID? Sending `sudo shutdown -h now` (over ssh) doesn't seem to stop the `tart run` process (even though you can't any longer ssh into the machine).

Are you talking about Linux VMs? Because both `shutdown -h now` and `halt` result in Tart process exiting for me when running macOS VMs.

**Comment by alexey-tereshenkov-oxb** [comment]
No, macOS one. Let me write up a little shell script to help reproducing!

**Comment by fkorotkov** [comment]
@edigaryev I wonder if it makes sense to add extra protection like checking there is something pingable via an IP that will be leased out to the VM.

Basically if you run and stop a VM there will be already an IP leased to it. So before running a VM we potentially can check if there is already something `ping`ing though the IP. It will definitely work for the case of running two VMs and I can't think if such a check can break something.

**Comment by fkorotkov** [comment]
Actually there is already a better check if tow VMs are running:

https://github.com/cirruslabs/tart/blob/048a5506df871aaddc412ddfabe2e52bec850e57/Sources/tart/Commands/Run.swift#L94-L97
